app/components/chat/chat-window.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
+  classNames: 'chat-window',
+  session: service(),
+
+  openChat: false,
+  windowSize: 'chat-closed',
+  contentChat: true,
+  messages: [],
+
+  actions: {
+    toggle: function() {
+      this.toggleProperty('openChat');
+      if(this.get('openChat')){
+        this.set('windowSize', 'chat-open');
+      } else {
+        this.set('windowSize', 'chat-closed');
+      }
+    },
+    sendMessage: function(message) {
+      if(this.get('messages.lastObject.userid') === this.get('session.user.id')){
```

Comment:
I am not a fan of concatenating the bodies of messages together, but I suppose it will do the job for now